app/components/Form/AnticipatedFundingAmountByFiscalYearArrayFieldTemplate.tsx

```diff
@@ -1,40 +1,12 @@
-import { WidgetProps } from "@rjsf/core";
-import { graphql, useFragment } from "react-relay";
-import CalculatedValueWidget from "./CalculatedValueWidget";
+import { ArrayFieldTemplateProps } from "@rjsf/core";
+import CalculatedValueWidget from "lib/theme/widgets/CalculatedValueWidget";
 
-const AnticipatedFundingAmountPerFiscalYearWidgetFragment = graphql`
-  fragment AnticipatedFundingAmountPerFiscalYearWidget_projectRevision on ProjectRevision {
-    formChangesByProjectRevisionId {
-      edges {
-        node {
-          anticipatedFundingAmountPerFiscalYear {
-            edges {
-              node {
-                anticipatedFundingAmount
-                fiscalYear
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-`;
-
-const AnticipatedFundingAmountPerFiscalYearWidget: React.FC<WidgetProps> = (
-  props
+const AnticipatedFundingAmountByFiscalYearArrayFieldTemplate = (
+  props: ArrayFieldTemplateProps
 ) => {
-  const query = useFragment(
-    AnticipatedFundingAmountPerFiscalYearWidgetFragment,
-    props.formContext.projectRevision
-  );
-
-  const anticipatedFunding = query.formChangesByProjectRevisionId?.edges[0]
-    ?.node.anticipatedFundingAmountPerFiscalYear?.edges[0]
-    ? [
-        ...query.formChangesByProjectRevisionId?.edges[0]?.node
-          .anticipatedFundingAmountPerFiscalYear?.edges,
-      ]
+  const anticipatedFunding = props.formContext
+    .anticipatedFundingAmountPerFiscalYear?.edges
```

Comment:
To display array fields, rjsf docs say to map `props.items.children`. However, in the editable budgets form, we don't have any items because anticipated funding per fiscal year is a calculated array based on fields in the milestone form. So instead I've used form context and made this "array field template" more like a widget. Not super happy with this implementation, suggestions very welcome